app/controllers/checklist_templates_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ChecklistTemplatesController < InheritedResources::Base
```

Comment:
Is this piggybacking off active admin? I think our intention is to remove active admin.